.github/workflows/deploy.yml

```
@@ -46,4 +46,40 @@ jobs:
         run: npm run build-storybook
       
       - name: deploy-storybook
-        run: npm run deploy-storybook
\ No newline at end of file
+        run: npm run deploy-storybook
+
+      - name: comment on pull request
+        uses: actions/github-script@v6
+        with:
+          script: |
+            try {
+              const issues = await github.rest.pulls.list({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                state: 'open',
+                head: `${context.repo.owner}:${context.ref.replace('refs/heads/', '')}`
+              });
+
+              const prNumber = context.issue.number || issues.data[0].number;
```

```suggestion
              const prNumber = context.issue?.number || issues.data[0].number;
```

앞쪽은 optional chaining해줘야 의도에 맞지 않을까 싶네요 ㅎ

---

넵 옵션널 체이닝 추가해주었습니다!